docs/use_cases.md

```diff
@@ -244,3 +244,27 @@ can be used to constrain an insertion target for instance.
 
 If an anchor constrains a dispersion, the dispersion will be intrachromosomal even if the SV is set as interchromosomal 
 (In this case, other unconstrained dispersions of the SV will then be interchromosomal). 
+
+
+### Example 5b - Inversion within segmental duplications (SDs)
+To simulate INVs that span the regions between SDs, from a BED file annotating the SD regions.
+
```

Comment:
switch to bedtools to compute the complement; rename regions_within_SDs.bed to regions_outside_SDs.bed or SD_comeplement.bed (since the original input bed contains the regions within SDs); remove overlap_region_type